apps/src/music/views/PatternAiPanel.tsx

```diff
@@ -395,7 +439,15 @@ const PatternAiPanel: React.FunctionComponent<PatternAiPanelProps> = ({
     );
     setGenerateState('generating');
     setGeneratingScanStep(1);
-  }, [currentValue, onChange, aiTemperature, stopPreview, playPreview]);
+    setGenerateCount(generateCount + 1);
```

Comment:
Super minor/optional, but in case you don't want to include `generateCount` as a dependency to this callback, you could also pass a function to increment the counter state like `setGenerateCount(count => count + 1)`

Reply:
Oh interesting.  Might do this in a follow-up.